### PR TITLE
Try again

### LINKED ISSUE #10: Try Again Page
- [ ] embedded try again image
- [ ] create Retry button
- [ ] making css
- [ ] render in index.js 
* note this file while adding in dom soon

### PR DESCRIPTION
**Pull request by marwa7med**
fixes #10 

### Summary
modified try again page files

### CODE REVIEW

File: public/css/tryAgain.css [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+	50% {
+		opacity: 0;
+	}
+	100% {
+		opacity: 1;
+	}
+}
+
+.pop-up {
+	animation: pop 0.3s ease 1s;
```
Comment (mhmdshorafa) [inline public/css/tryAgain.css:35]: please ,, try to keep consistent with indentation ,, and choose if it is 4 spaces or 2 spaces

**Review by mhmdshorafa — COMMENTED** [review]
please keep on track with indentation consistency and choose one format ,, 2 spaces or 4 spaces

File: public/css/tryAgain.css (hunk 1 above)
Comment (marwa7med) [reply public/css/tryAgain.css:35]: ok , i always make beautify

Comment (mhmdshorafa) [reply public/css/tryAgain.css]: you should keep to the rules of eslint instead of using beautify

**Pull request merged**